### PR TITLE
Don't lint braces on string interpolation if identifier starts with $.

### PR DESCRIPTION
**Pull request by alexeieleusis**
#234

### CODE REVIEW

File: lib/src/rules/unnecessary_brace_in_string_interp.dart [hunk 1]
```diff
@@ -55,8 +55,10 @@ class Visitor extends SimpleAstVisitor {
     var expressions = node.elements.where((e) => e is InterpolationExpression);
     for (InterpolationExpression expression in expressions) {
       if (expression.expression is SimpleIdentifier) {
+        SimpleIdentifier identifier = expression.expression;
         Token bracket = expression.rightBracket;
-        if (bracket != null && !isIdentifierPart(bracket.next)) {
+        if (bracket != null && !isIdentifierPart(bracket.next) &&
+            identifier.name.indexOf(r'$') == -1) {
```
Comment (bwilkerson) [inline lib/src/rules/unnecessary_brace_in_string_interp.dart]: This doesn't need to be a raw string.

**Comment by bwilkerson** [comment]
lgtm

File: lib/src/rules/unnecessary_brace_in_string_interp.dart (hunk 1 above)
Comment (alexeieleusis) [inline lib/src/rules/unnecessary_brace_in_string_interp.dart]: fixed

**Comment by pq** [comment]
LGTM!

**Pull request merged**